.github/supporting-files/ci/partitions/partition0.txt

```diff
@@ -3,6 +3,3 @@
 @osl/osl-data-index-postgresql-image
 @osl/osl-jobs-service-ephemeral-image
 @osl/osl-jobs-service-postgresql-image
-@osl/osl-management-console-image
```

Comment:
@ricardozanini I had to reduce the partitioning here to fix the Windows CI failure:
````
error: The command line is too long.

   errno: -1
   code: "1"
 syscall: "spawnSync"
   path: "C:\Windows\system32\cmd.exe"
```